upgrade.php

```diff
@@ -201,33 +248,22 @@
     $passport = shell_exec('php artisan passport:install');
     echo $passport;
 } else {
-    echo "- OAuth keys detected. Skipping passport install.\n\n";
+    echo "√ OAuth keys detected. Skipping passport install.\n\n";
 }
 
 
 echo "--------------------------------------------------------\n";
-echo "Step 9: Caching routes and config:\n";
-echo "--------------------------------------------------------\n\n";
-$config_cache = shell_exec('php artisan config:cache');
-$route_cache = shell_exec('php artisan route:cache');
-echo '-- '.$config_cache;
-echo '-- '.$route_cache;
-echo "\n";
```

Comment:
I wonder if we should even go one step further, and *actively* remove their config-cache and route-caches via Artisan? Will stale caches there mess us up at all?

Reply:
They *shouldn't* mess us up there, and if the artisan command doesn't handle it, it's gonna be a permissions issue that the script get stuck on removing them manually anyway.

Reply:
no problemo.